lib/voice/VoiceConnection.js

```diff
@@ -70,7 +70,7 @@ converterCommand.pickCommand = function pickCommand() {
 * @prop {Number} current.pausedTimestamp The timestamp of the most recent pause
 * @prop {Number} current.playTime How long the current stream has been playing for, in milliseconds
 * @prop {Number} current.startTime The timestamp of the start of the current stream
-* @prop {String} id The ID of the voice connection (guild ID)
```

Comment:
having `(guild ID)` in parenthesis seems clearer than "the guild id of the voice connection"